### ISSUE TITLE
The Swoosh instrument demo not result same sound in R11 and newer version e.g. v14.7.77

### ISSUE THREAD

**Issue by lancety** [Tonejs/Tone.js#1032]
**Describe the bug**
I found another preset site here https://www.guitarland.com/MusicTheoryWithToneJS/Presets-gh-pages/  tone.js ver R11
and worked out a prefered swoosh sound json. but it does not sound same in http://tonejs.github.io/Presets/  tone.js ver 14.7.77
the official site demo sound does not feels right as "swoosh".  I also made my local test, tried these 2 versions, results are consistent.
So there must be something wrong.

**To Reproduce**
go to above 2 sites, select NoiseSynth - Swoosh, keep the default json, click "disconnect" on the R11 version site.  
press key and listen the difference.

**Expected behavior**
offcial (github) demo should sound as expected like the R11 version site.

**What I've tried**
created my local file and tested above 2 versions, same issue.

**Additional context**
This NoiseSynth Swoosh is a very suitable one for my game sound effect, really hope it can be fixed. Thanks for someone's help :)

**Comment by lancety** [comment]
If I set the envelope - attack to '3' seconds, in R11 demo, the noise came in at around 1.5 to 2 second,    in the github demo with same change , the noise directly came in when press the key which clearly is wrong I believe.

**Comment by lancety** [comment]
thanks @tambien ,  the attackCurve made the main sound difference compare to R11, I also reduced the decay and release time by 50%, from 0.5 to 0.25 to make it sounds same duration. 
In my test case, I also used the 'AutoWah' effect and reduce 'octaves' from 4 to 1 to make it sounds similar like R11.
Now the result is very close to the R11 version 👍  

About the compatibility issue, as a developer I have no problem to update code to new API, but I guess there are many developer like me who used this lib but dont have that much knowledge about music and sound, big changes of the sound output is a bigger challenge than API compatibility.   So I took several days to study music and sound, and the R11 demo site is one of the most useful tutorial. 

This lib is the most suitable one for my game project audio. Thanks all contributors for making this awesome lib. 🥇 

BTW, here is the whole 29 tutorials explaining music theory concepts by using tone.js R11, for developers like me :D   https://www.guitarland.com/MusicTheoryWithToneJS/TonejsSetup.html
